tests/hbacrule/test_hbacrule_categories.yml

```diff
@@ -109,6 +109,66 @@
     register: result
     failed_when: result.changed
 
+  - name: Ensure `user` cannot be added if usercategory is `all`.
+    ipahbacrule:
+      ipaadmin_password: SomeADMINpassword
+      name: allusers
+      user: shouldfail01
+      usercategory: "all"
+    register: result
+    ignore_errors: yes
```

Comment:
Please do not use ignore_errors as this will ignore all errors, also a backtrace.

Reply:
Fixed.